### PR TITLE
docs(other): correct **/README.md

### PR DESCRIPTION
**Pull request by Elweyn**
### Issues

- None

### Todo

- [X] Contraint to Constraint
- [X] Indexes to Indices
- [X] Add migrations up to docus

### CODE REVIEW

**Review by mahula — APPROVED** [review]

**Review by Mogge — APPROVED** [review]

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -190,7 +190,7 @@ For Docker compose `up` or `build` commands, you can use our Apple M1 override f
 
 # for development
 $ docker compose -f docker-compose.yml -f docker-compose.override.yml -f docker-compose.apple-m1.override.yml up
-# only once: init admin user and create indexes and contraints in Neo4j database
+# only once: init admin user and create indices and constraints in Neo4j database
```
Comment (Tirokk) [inline CONTRIBUTING.md:193]: Indexes is the correct namimg, see headline of:
https://neo4j.com/docs/cypher-manual/current/indexes-for-search-performance/

```suggestion
# only once: init admin user and create indexes and constraints in Neo4j database
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -199,7 +199,7 @@ $ docker compose exec backend yarn db:seed
 
 # for production
 $ docker compose -f docker-compose.yml -f docker-compose.apple-m1.override.yml up
-# only once: init admin user and create indexes and contraints in Neo4j database
+# only once: init admin user and create indices and constraints in Neo4j database
```
Comment (Tirokk) [inline CONTRIBUTING.md:202]: same same

```suggestion
# only once: init admin user and create indexes and constraints in Neo4j database
```

File: README.md [hunk 3]
```diff
@@ -139,6 +139,7 @@ Prepare database once before you start by running the following command in a sec
 ```bash
 # in main folder while docker-compose is up
 $ docker-compose exec backend yarn run db:migrate init
+$ docker-compose exec backend yarn run db:migrate up
```
Comment (Tirokk) [inline README.md:142]: Without dash is the modern command, see:
https://docs.docker.com/compose/reference/

```suggestion
$ docker compose exec backend yarn run db:migrate init
$ docker compose exec backend yarn run db:migrate up
```

File: backend/README.md [hunk 4]
```diff
@@ -126,8 +135,10 @@ To reset the database run:
 $ docker exec backend yarn run db:reset
 # you could also wipe out your neo4j database and delete all volumes with:
 $ docker-compose down -v
-# if container is not running, run this command to set up your database indeces and contstraints
+# if container is not running, run this command to set up your database indices and constraints
```
Comment (Tirokk) [inline backend/README.md:138]: same same

```suggestion
# if container is not running, run this command to set up your database indexes and constraints
```

File: backend/README.md [hunk 5]
```diff
@@ -126,8 +135,10 @@ To reset the database run:
...
 $ docker exec backend yarn run db:migrate init
+# And then upgrade the indices and const
```
Comment (Tirokk) [inline backend/README.md:140]:
```suggestion
# And then upgrade the indexes and const
```

File: deployment/DOCKER_MORE_CLOSELY.md [hunk 6]
```diff
@@ -24,7 +24,7 @@ $ docker compose -f docker-compose.yml -f docker-compose.apple-m1.override.yml u
 # for production testing Docker images from DockerHub
 $ docker compose -f docker-compose.ocelotsocial-branded.yml -f docker-compose.apple-m1.override.yml up
 
-# only once: init admin user and create indexes and contraints in Neo4j database
+# only once: init admin user and create indices and constraints in Neo4j database
```
Comment (Tirokk) [inline deployment/DOCKER_MORE_CLOSELY.md:27]:
```suggestion
# only once: init admin user and create indexes and constraints in Neo4j database
```

File: neo4j/README.md [hunk 7]
```diff
@@ -55,15 +55,15 @@ Start Neo4J and confirm the database is running at [http://localhost:7474](http:
 
 Here we describe some rarely used Cypher commands for Neo4j that are needed from time to time:
 
-### Index And Contraint Commands
+### Index And Constraint Commands
 
-If indexes or constraints are missing or not set correctly, the browser search will not work or the database seed for development will not work.
+If indices or constraints are missing or not set correctly, the browser search will not work or the database seed for development will not work.
```
Comment (Tirokk) [inline neo4j/README.md:60]:
```suggestion
If indexes or constraints are missing or not set correctly, the browser search will not work or the database seed for development will not work.
```

File: neo4j/README.md [hunk 8]
```diff
@@ -55,15 +55,15 @@ Start Neo4J and confirm the database is running at [http://localhost:7474](http:
...
 
-The indexes and constraints of our database are set in `backend/src/db/migrate/store.js`.
+The indices and constraints of our database are set in `backend/src/db/migrate/store.js`.
```
Comment (Tirokk) [inline neo4j/README.md:62]:
```suggestion
The indexes and constraints of our database are set in `backend/src/db/migrate/store.js`.
```

File: neo4j/README.md [hunk 9]
```diff
@@ -55,15 +55,15 @@ Start Neo4J and confirm the database is running at [http://localhost:7474](http:
...
 This is where the magic happens.
 
 It's called by our `prod:migrate init` command.
-This command initializes the Admin user and creates all necessary indexes and constraints in the Neo4j database.
+This command initializes the Admin user and creates all necessary indices and constraints in the Neo4j database.
```
Comment (Tirokk) [inline neo4j/README.md:66]:
```suggestion
This command initializes the Admin user and creates all necessary indexes and constraints in the Neo4j database.
```

File: neo4j/README.md [hunk 10]
```diff
@@ -98,34 +98,34 @@ On a server with Kubernetes cluster:
 $ kubectl -n default exec -it $(kubectl -n default get pods | grep ocelot-backend | awk '{ print $1 }') -- /bin/sh -c "yarn prod:migrate init"
 ```
 
-***Cypher commands to show indexes and constraints***
+***Cypher commands to show indices and constraints***
```
Comment (Tirokk) [inline neo4j/README.md:101]:
```suggestion
***Cypher commands to show indexes and constraints***
```

File: neo4j/README.md [hunk 11]
```diff
@@ -98,34 +98,34 @@ On a server with Kubernetes cluster:
...
 
 ```bash
 # in browser command line or cypher shell
 
-# show all indexes and constraints
+# show all indices and constraints
```
Comment (Tirokk) [inline neo4j/README.md:106]:
```suggestion
# show all indexes and constraints
```

File: neo4j/README.md [hunk 12]
```diff
@@ -98,34 +98,34 @@ On a server with Kubernetes cluster:
...
 $ :schema
 
-# show all indexes
+# show all indices
```
Comment (Tirokk) [inline neo4j/README.md:109]:
```suggestion
# show all indexes
```

File: neo4j/README.md [hunk 13]
```diff
@@ -98,34 +98,34 @@ On a server with Kubernetes cluster:
...
 $ CALL db.indexes();
 
 # show all constraints
 $ CALL db.constraints();
 ```
 
-***Cypher commands to create and drop indexes and constraints***
+***Cypher commands to create and drop indices and constraints***
```
Comment (Tirokk) [inline neo4j/README.md:116]:
```suggestion
***Cypher commands to create and drop indexes and constraints***
```

File: neo4j/README.md [hunk 14]
```diff
@@ -98,34 +98,34 @@ On a server with Kubernetes cluster:
...
 
 ```bash
 # in browser command line or cypher shell
 
-# create indexes
+# create indices
```
Comment (Tirokk) [inline neo4j/README.md:121]:
```suggestion
# create indexes
```

File: neo4j/README.md [hunk 15]
```diff
@@ -98,34 +98,34 @@ On a server with Kubernetes cluster:
...
 $ CALL db.index.fulltext.createNodeIndex("post_fulltext_search",["Post"],["title", "content"]);
 $ CALL db.index.fulltext.createNodeIndex("user_fulltext_search",["User"],["name", "slug"]);
 $ CALL db.index.fulltext.createNodeIndex("tag_fulltext_search",["Tag"],["id"]);
 
 # drop an index
 $ DROP CONSTRAINT ON ( image:Image ) ASSERT image.url IS UNIQUE
 
-# drop all indexes and constraints
+# drop all indices and constraints
```
Comment (Tirokk) [inline neo4j/README.md:129]:
```suggestion
# drop all indexes and constraints
```

**Review by Tirokk — CHANGES_REQUESTED** [review]
I have some suggestions.

PS: An indice is the small number behind a mathematical variable or coeffecent. An index is a kind of a register.

File: backend/README.md [hunk 16]
```diff
@@ -81,8 +81,7 @@ More details about our GraphQL playground and how to use it with ocelot.social c
 
 ### Database Indices and Constraints
 
-Database indices and constraints need to be created when the database and the
-backend is running:
+Database indices and constraints need to be created and upgraded when the database and the backend are running:
```
Comment (Tirokk) [inline backend/README.md:84]:
```suggestion
Database indexes and constraints need to be created and upgraded when the database and the backend are running:
```

**Review by Tirokk — CHANGES_REQUESTED** [review]
Sorry, I found one additional.
Then it's okay.

File: backend/README.md [hunk 17]
```diff
@@ -81,8 +81,7 @@ More details about our GraphQL playground and how to use it with ocelot.social c
 
 ### Database Indices and Constraints
```
Comment (Elweyn) [inline backend/README.md:82]:
```suggestion
### Database Indexes and Constraints
```

**Pull request merged**